lib/core-data/groups.js

```diff
@@ -94,13 +94,15 @@ export function getSettingsFields(data, schema) {
   } else {
     // look for all fields with `_display: settings`
     // note: no sections if there isn't a manual settings group
-    return { fields: _.reduce(data, function (fields, fieldData, fieldName) {
-      if (_.get(schema, `${fieldName}.${displayProp}`) === 'settings') {
-        fields[fieldName] = fieldData;
-      }
-
-      return fields;
-    }, {}), sections: [] };
+    return {
+      fields: _.reduce(schema, (fields, fieldData, fieldName) => {
```

Comment:
this `fieldData` is now the `fieldSchema`, so you don't need to do a deep get on the line after